apply/default_applier.go

```diff
@@ -131,6 +138,25 @@ var ActionFuncMap = map[ActionName]func(*DefaultApplier) error{
 	CleanFS: func(applier *DefaultApplier) error {
 		return applier.FileSystem.Clean(applier.ClusterDesired)
 	},
+	PluginConfig: func(applier *DefaultApplier) error {
+		return applier.Plugin.Dump(applier.ClusterDesired.GetAnnotationsByKey(common.ClusterfileName))
+	},
+	PluginPhasePreInitRun: func(applier *DefaultApplier) error {
+		plugins := applier.Plugin.GetPhasePlugin("PreInit")
```

Comment:
I think don't need to get phase plugin because some plugin config doesn't config the `on` field. 

You just load all the plugin lists and Run them in every phase, because the plugin itself will consider run or not in the specific phase.